src/vehicle.cpp

```diff
@@ -1597,26 +1599,44 @@ void Vehicle::UpdatePosition()
 	UpdateVehicleTileHash(this, false);
 }
 
-/**
- * Update the vehicle on the viewport, updating the right hash and setting the
- *  new coordinates.
- * @param dirty Mark the (new and old) coordinates of the vehicle as dirty.
- */
-void Vehicle::UpdateViewport(bool dirty)
+/*
+ * Update the bounding box co-ordinates of the vehicle
+ * @param update_cache Update the cached values for previous co-ordinate values
+*/
+void Vehicle::UpdateBoundingBoxCoordinates(bool update_cache)
 {
 	Rect new_coord;
 	this->sprite_cache.sprite_seq.GetBounds(&new_coord);
 
 	Point pt = RemapCoords(this->x_pos + this->x_offs, this->y_pos + this->y_offs, this->z_pos);
-	new_coord.left   += pt.x;
-	new_coord.top    += pt.y;
-	new_coord.right  += pt.x + 2 * ZOOM_LVL_BASE;
+	new_coord.left += pt.x;
+	new_coord.top += pt.y;
+	new_coord.right += pt.x + 2 * ZOOM_LVL_BASE;
```

Comment:
Editor decided to "fix" alignment here ;)

Reply:
I spotted this on an earlier diff and failed to go back and fix it - thanks for picking up, should be fixed now :)